octoprint_findmymrbeam/__init__.py

```diff
@@ -139,10 +163,29 @@ def is_online_gif(self, secret):
 
 		# send a transparent 1x1 px gif
 		import base64
-		response = flask.make_response(bytes(base64.b64decode("R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7")))
+		response = flask.make_response(
+			bytes(base64.b64decode("R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7")))
 		response.headers["Content-Type"] = "image/gif"
 		return response
 
+	@octoprint.plugin.BlueprintPlugin.route("/<secret>.json", methods=["GET", "OPTIONS"])
+	def get_device_info(self, secret):
```

Comment:
Consider a different name, this does not seem to describe the function, or please add a docstring.

`get_device_info` is also used in the mrb plugin, doesn't seem to be related.